content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -99,6 +164,7 @@ Create a new FamilyMemberHistory.
 _Implementation Notes_
 
 * Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Conditions can only be added through the PUT operation.
```

Comment:
let's keep this specific to creates...

Conditions cannot be added while creating the FamilyMemberHistory.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/728/commits/9b6b77e5205ad78964a0ac908ff67562395e225f#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L167